node/src/chain_spec/neumann.rs

```diff
@@ -307,6 +310,7 @@ fn testnet_genesis(
 			phantom: Default::default(),
 		},
 		parachain_system: Default::default(),
+		polkadot_xcm: PolkadotXcmConfig { safe_xcm_version: Some(SAFE_XCM_VERSION) },
```

Comment:
i'm guessing this is just for us to decide on whether we're using version 0-2, right? i'm curious what it's actually used for in polkadot. the only place where i could find it being used was https://github.com/paritytech/polkadot/blob/master/xcm/pallet-xcm/src/lib.rs#L1224 and i have no idea what it's doing there

Reply:
that function checks the receiving network for the supported version and then returns the minimum version number between the sending and receiving networks.  pretty much enforcing backwards compatibility but not forwards.  from the this article (https://polkadot.network/blog/xcm-part-two-versioning-and-compatibility/):
"It must therefore be the responsibility of the sending network to ensure that the message it sends is capable of being interpreted by the receiving network. In concrete terms, the version of XCM used for the message must be no newer than the version of XCM that the receiving network supports."